test/autoscaling.test.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+const assume = require('assume');
+const async = require('async');
+const nock = require('nock');
+
+const common = require('./common');
+const beforeTesting = common.beforeTesting;
+const resourceName = 'test-name';
+
+describe('lib.autoscaling', () => {
+  describe('.autoscaling', () => {
+    const path = `/apis/autoscaling/v1/namespaces/${ common.currentName }/horizontalpodautoscalers`;
+    const resourcePath = `${ path }/${ resourceName }`;
+    const horizontalAutoscaleObj = {
```

Comment:
An almost identical value for a `HorizontalPodAutoscaler` resource is defined several times (*e.g*., https://github.com/godaddy/kubernetes-client/pull/152/files#diff-179fe5b2cb9d0365ec10002d994f44d2R41 and https://github.com/godaddy/kubernetes-client/pull/152/files#diff-179fe5b2cb9d0365ec10002d994f44d2R92).

Can you reduce that duplication?

Reply:
Duplicates removed...